_chapters/chapter_04.md

```diff
@@ -705,43 +721,43 @@ layout: rails_tutorial
 
       At the end, you should see a nil description.
 
-      ```
+      ```shell
       => #<Book id: 1, title: "why's (poignant) Guide to Ruby", ..., description: nil>
       ```
 
       `description` is nil because you just added it.
 
-  1.  You can give `my_first_book` a description by running
+  1.  You can give your first book a description by running
```

Comment:
Can we change this back to `my_first_book`? Two steps ago we told them to assign the first book to `my_first_book`.

Using variables is a subtle nudge towards objects and instances. It also matches what we do in the controllers.